### PR TITLE
Add order gallery

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js [hunk 1]
```diff
@@ -24,26 +33,64 @@ class MediaListView extends mix(AbstractDataTableView).with(UrlPaginateViewMixin
...
+            this._collection.each(function(media) {
+                let $template = $(context._renderTemplate(templateFile, {media: media}));
+                context._createDeleteCheckbox.apply(mediaList, [$('.delete-button', $template), null, media]);
+                mediaList.append($template);
+            });
+        }
         $(".table-responsive", this.$el).html(mediaList);
+
+        if($(".header-results-order", this.$el).length == 0) {
+            $(".header-results", this.$el).append('<div class="header-results-order">');
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js]: use property domsettings

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js [hunk 2]
```diff
@@ -77,15 +124,21 @@ class MediaListView extends mix(AbstractDataTableView).with(UrlPaginateViewMixin
     /**
      * @inheritDoc
      */
-    getColumnsDefinition() {
-        return [];
+    generateUrlUpdatePage(page) {
+       return Backbone.history.generateUrl('listMedia', {page : page});
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js:136]: update paginate not work

Comment (itkg-nanne) [inline MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js]: I'm not sure to understand. If I got sorting on page 1, clicking on page 2, keep the sort.

**Comment by itkg-nanne** [comment]
update done @alavieille

File: MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js (hunk 2 above)
Comment (veolys-nanne) [reply MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js:136]: not working on master. Fixed.

**Comment by veolys-nanne** [comment]
update done, @alavieille

**Comment by alavieille** [comment]
:+1:

**Pull request merged**